Platform/TigerlakeBoardPkg/AcpiTables/CpuSsdt/CpuSsdt.asl

```diff
@@ -415,40 +415,41 @@ Scope (\_SB.PR00)
     //
     // Operation region for Smi port access
     //
-    OperationRegion (SMIP, SystemIO, 0xB2, 1)
-    Field (SMIP, ByteAcc, NoLock, Preserve)
-    {
-      IOB2, 8
-    }
-
+// BIOS ACPI table removed
```

Comment:
For all commented ASL code, let us just remove it to make it clean.

Reply:
Sure, removed.